src/main/kotlin/gdx/liftoff/data/platforms/Lwjgl3.kt

```diff
@@ -144,25 +144,24 @@ class Lwjgl3GradleFile(val project: Project) : GradleFile(Lwjgl3.ID) {
     addDependency("com.badlogicgames.gdx:gdx-platform:\$gdxVersion:natives-desktop")
   }
 
-  override fun getContent(): String = """buildscript {
+  // language=groovy
+  override fun getContent(): String = """
+import io.github.fourlastor.construo.Target
+
+buildscript {
   repositories {
     gradlePluginPortal()
   }
   dependencies {
-// using jpackage only works if the JDK version is 14 or higher.
-// your JAVA_HOME environment variable may also need to be a JDK with version 14 or higher.
-    if (JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_14)) {
-      classpath "org.beryx:badass-runtime-plugin:1.13.0"
-    }
     if(enableGraalNative == 'true') {
       classpath "org.graalvm.buildtools.native:org.graalvm.buildtools.native.gradle.plugin:0.9.28"
     }
   }
 }
-${if (project.rootGradle.plugins.contains("kotlin")) "apply plugin: 'org.jetbrains.kotlin.jvm'\n" else ""}
-if (JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_14)) {
-  apply plugin: 'org.beryx.runtime'
+plugins {
+  id "io.github.fourlastor.construo" version "1.1.1"
 }
+${if (project.rootGradle.plugins.contains("kotlin")) "apply plugin: 'org.jetbrains.kotlin.jvm'\n" else ""}
 else {
```

Comment:
I don't see a corresponding `if` for this `else` block. Liftoff itself may be able to compile and run, but projects it creates might not be valid if there's an error like a mismatched `else`.

Reply:
This was an oversight on my end, it should always use the `application` plugin now [ba49762](https://github.com/libgdx/gdx-liftoff/pull/155/commits/ba497621119d399b74ad3a1a3d1ebad615bdf139)